default.json

```diff
@@ -126,6 +126,18 @@
     {
       "matchRepositories": ["balena-io/balena-ui"],
       "commitBody": "Update {{depName}}\n\nChangelog-entry: Update {{depName}}\nChange-type: patch"
+    },
+    {
+      "matchPackagePatterns": ["^husky$"],
+      "matchUpdateTypes": ["major"],
+      "postUpgradeTasks": {
+        "commands": [
+          "jq '.scripts.prepare |= if . == \"node -e \\\"try { require(\\'husky\\').install() } catch (e) {if (e.code !== \\'MODULE_NOT_FOUND\\') throw e}\\\"\" then \"node -e \\\"try { (await import(\\'husky\\')).default() } catch (e) { if (e.code !== \\'ERR_MODULE_NOT_FOUND\\') throw e }\\\" --input-type module\" else . end' package.json > package.json.tmp && mv package.json.tmp package.json",
```

Comment:
This will work great for a bunch of cases however it'll fall down on cases where we also need to use the prepare script for more traditional cases, eg https://github.com/balena-io-modules/typed-error/blob/v3.2.2/package.json#L19

Reply:
We can add additional matches if this one works for the simplest use case!